receipt_label/receipt_label/merchant_resolution/chroma.py

```
@@ -0,0 +1,279 @@
+                    phone_ok = False
+                else:
+                    neighbor_digits = neighbor_phone
+                    if not neighbor_digits and get_neighbor_phones:
+                        try:
+                            nid = (
+                                str((md or {}).get("image_id") or ""),
+                                int((md or {}).get("receipt_id") or 0),
+                            )
+                            fetched = get_neighbor_phones(nid) or []
+                            neighbor_digits = fetched[0] if fetched else ""
+                        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)